### PR TITLE
Revert "Revert "remove na option added""

### PR DESCRIPTION
**Pull request by antagomir**
Reverts microbiome/mia#203

### CODE REVIEW

File: R/summaries.R [hunk 1]
```diff
@@ -13,7 +13,10 @@
 #'   median or prevalence. Default is 'mean'.
 #'
 #' @param abund_values A \code{character} value to select an
-#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assayNames}}
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assayNames}} 
+#'   
+#' @param na.rm For \code{getTopTaxa} logical argument for calculation method 
+#'              specified to argument \code{method}. Default is TRUE.
```
Comment (antagomir) [inline R/summaries.R:19]: These "methods" functions have also other possible arguments available, besides `na.rm`. 

We could allow the user to pass any of those available arguments through the "..." mechanism, I think we should allow this unless there are specific reason to not allow it. Then we could just remove the na.rm param description and state in "..." argument documentation (line 21 currently) that any other arguments available through \code{method} can be passed here, and cite the documentation of those functions for more details.

File: DESCRIPTION [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Package: mia
 Type: Package
-Version: 1.3.13
+Version: 1.3.14
```
Comment (antagomir) [inline DESCRIPTION:3]: should be checked before merge.

File: R/summaries.R (hunk 1 above)
Comment (antagomir) [reply R/summaries.R:19]: @microsud

Comment (microsud) [reply R/summaries.R:19]: If we add `...` then every time we may need to specify `na.rm` in the function. By using `na.rm=TRUE` as default IMO is more user friendly.  Other arguments are never used, at least I have never used it for this particular function which returns specific output.

Comment (antagomir) [reply R/summaries.R:19]: Fair point. I am ok with this then.

**Review by antagomir — COMMENTED** [review]
Can you update the NEWS file?

**Comment by microsud** [comment]
@antagomir checks are clear, if you agree we merge this?

**Comment by microsud** [comment]
Thanks!

**Comment by FelixErnst** [comment]
What was this about?

**Pull request merged**